docs/new-feature-tutorial.rst

```diff
@@ -2,84 +2,178 @@
 New Feature Tutorial
 ====================
 
-.. attention::
-   This tutorial is an unfinished work -- contributions welcome!
+The changes needed to add a new feature will vary, of course, but this document
+provides a general outline of what you may need to do, as well as an example of
+the specific steps needed to add a new feature;
```

Comment:
Maybe extend this to be explicit that this focuses on what's involved in adding a new option that is dynamically synced through the data system to all browsers the user may have open in real-time?